gslib/commands/cp.py

```diff
@@ -631,147 +615,141 @@
                  - Source size.
                  - Bytes transferred.
                  - MD5 hash.
-                 - UTC date and time transfer was started in ISO 8601 format.
-                 - UTC date and time transfer was completed in ISO 8601 format.
+                 - UTC date and transfer start time in ISO 8601 format.
```

Comment:
I think you'll want to keep the "date" and "time" portion of these lines grouped together. Read literally, the proposed wording could "The current UTC date, as well as the time the transfer was started/completed"